src/events/guildMemberRemove.ts

```diff
@@ -76,5 +77,17 @@ export function makeHandler({ database, challonge }: CommandSupport) {
 				}
 			})
 			.catch(logger.error);
+		const manualParticipants = await ManualParticipant.getRepository()
+			.createQueryBuilder()
+			.innerJoinAndSelect(
+				"ManualParticipant.tournament",
+				"T",
+				"(T.status = 'preparing' OR T.status = 'in progress') AND T.owningDiscordServer = :server AND Participant.discordId = :playerId",
+				{ server: server.id, playerId: member.id }
+			)
+			.getMany();
+		for (const participant of manualParticipants) {
+			await dropPlayer(participant.tournament, participant, member);
+		}
```

Comment:
Last thing, wrap this in a try-catch with logger error like the previous transaction, since this handler is live instantly, so any errors do not crash the bot.